src/utils/oauth2_util.py

```diff
@@ -1,8 +1,47 @@
 from datetime import datetime, timedelta
 
+from src.constants.oauth2 import Roles, Scopes
+from src.constants.http import HttpMethods
+from src.lib.entities.secured_http_request_uri import SecuredHttpRequestUrl
+from src.lib.entities.endpoint_roles_scopes import EndpointRolesScopes
 import bcrypt
 import jwt
 
+from src.exceptions.exceptions import UnAuthorizedEndpoint
+
+ENDPOINT_ROLES_MAP = {
+    SecuredHttpRequestUrl(
+        path="/chefs", method=HttpMethods.GET.value
+    ): EndpointRolesScopes(
+        roles=[Roles.ADMINISTRATOR.value], scopes=[Scopes.READ_WRITE.value]
```

Comment:
I think there should be 2 scopes, one for writing and the other one for reading

Reply:
done.